packages/react/src/Label/Label.tsx

```diff
@@ -0,0 +1,104 @@
+import clsx from 'clsx'
+import React, {forwardRef, type Ref} from 'react'
+import {Text} from '../Text'
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/label/colors-with-modes.css'
+import type {BaseProps} from '../component-helpers'
+import styles from './Label.module.css'
+
+export const LabelColors = [
+  'default',
+  'blue',
+  'blue-purple',
+  'coral',
+  'green',
+  'green-blue',
+  'gray',
+  'indigo',
+  'lemon',
+  'lime',
+  'orange',
+  'pink',
+  'pink-blue',
+  'purple',
+  'purple-red',
+  'red',
+  'red-orange',
+  'teal',
+  'yellow'
```

Comment:
If none of these values were suitable for users and someone needed an escape-hatch, what's the recommended approach to customising the color?

I see we have tokens `--brand-Label-color-start` and `--brand-Label-color-end`, that get reassigned based on the color value mapping in css, but to avoid app devs overriding the other values here, or introducing global overrides, should we consider introducing a `custom` value here, which maps to something like `--brand-Label-color-custom`?

Reply:
Okay, so just to be clear.. if i want to have one new color variation (say pink-orange) of a Label in an app, purely as a one-off.. am i still going to be able to use all the other built-in color values as normal? or will that custom color override prevent the originals from working correctly?

Reply:
You could define that one-off color by either overriding the  `--brand-Label-color-start` and `--brand-Label-color-end` in selecting the specific Label class or creating a custom one with those changes and adding it to the Label,  and that won't affect the rest of the Label colors on a page

Example modifying the first Label colors in the playground to pink-orange:

<img width="1456" alt="Screenshot 2023-04-13 at 12 08 13" src="https://user-images.githubusercontent.com/912236/231727115-e2f7f921-f877-41b3-87cc-1c802bce9e66.png">

Reply:
> and that won't affect the rest of the Label colors on a page

👍 works for me. thanks for clarifying @josepmartins